libgit2-sys/lib.rs

```diff
@@ -1754,6 +1755,17 @@ git_enum! {
     }
 }
 
+#[repr(C)]
+pub struct git_stash_save_options {
+    pub version: c_uint,
+    pub flags: c_uint,
```

Comment:
It looks like this is `uint32_t`

```suggestion
    pub flags: u32,
```